parachain/runtime/interlay/src/lib.rs

```diff
@@ -556,8 +556,8 @@ impl pallet_multisig::Config for Runtime {
 
 parameter_types! {
     pub const ProposalBond: Permill = Permill::from_percent(5);
-    pub ProposalBondMinimum: Balance = 5;
-    pub ProposalBondMaximum: Option<Balance> = None;
+    pub ProposalBondMinimum: Balance = 5 * UNITS;
+    pub ProposalBondMaximum: Balance = 25 * UNITS;
```

Comment:
Why does this compile? As far as I can see the type in treasury is still `type ProposalBondMaximum: Get<Option<BalanceOf<Self, I>>>;`?